### ISSUE TITLE
Inline editor doesn't work for JSONB columns

### ISSUE THREAD

**Issue by izonder** [dbeaver/dbeaver#7438]
#### System information: 
- Ubuntu 18.04 (4.15.0-69-generic #78-Ubuntu SMP Wed Nov 6 11:30:24 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux)
- DBeaver 6.3.0

#### Connection specification:
- Postgres 9.6
- org.postgresql-42.2.5
- org.postgis-jdbc-2.2.1

#### Describe the problem you're observing:

With a double-click on a cell from JSONB column, there is no reaction. With other types, the inline editor works properly.

![Selection_171](https://user-images.githubusercontent.com/1416966/71084632-03c0cf00-2196-11ea-83fb-4b5aec0f3219.png)
![Selection_172](https://user-images.githubusercontent.com/1416966/71084644-0a4f4680-2196-11ea-8f04-d2200f69da74.png)
![Selection_170](https://user-images.githubusercontent.com/1416966/71084657-0f13fa80-2196-11ea-8ea7-499058f8e772.png)

#### Steps to reproduce, if exist:

```sql
DROP TABLE public.tjsonb;

CREATE TABLE public.tjsonb (
	pk_id integer NOT NULL,
	payload jsonb NOT NULL,
	CONSTRAINT tjsonb_pk PRIMARY KEY (pk_id)
);

GRANT ALL ON TABLE public.tjsonb TO postgres;

INSERT INTO public.tjsonb
VALUES (1, '{
  "name": "South America",
  "summary": "South America (Spanish: América del Sur, Sudamérica or  \nSuramérica; Portuguese: América do Sul; Quechua and Aymara:  \nUrin Awya Yala; Guarani: Ñembyamérika; Dutch: Zuid-Amerika;  \nFrench: Amérique du Sud) is a continent situated in the  \nWestern Hemisphere, mostly in the Southern Hemisphere, with  \na relatively small portion in the Northern Hemisphere.  \nThe continent is also considered a subcontinent of the  \nAmericas.[2][3] It is bordered on the west by the Pacific  \nOcean and on the north and east by the Atlantic Ocean;  \nNorth America and the Caribbean Sea lie to the northwest.  \nIt includes twelve countries: Argentina, Bolivia, Brazil,  \nChile, Colombia, Ecuador, Guyana, Paraguay, Peru, Suriname,  \nUruguay, and Venezuela. The South American nations that  \nborder the Caribbean Sea—including Colombia, Venezuela,  \nGuyana, Suriname, as well as French Guiana, which is an  \noverseas region of France—are also known as Caribbean South  \nAmerica. South America has an area of 17,840,000 square  \nkilometers (6,890,000 sq mi). Its population as of 2005  \nhas been estimated at more than 371,090,000. South America  \nranks fourth in area (after Asia, Africa, and North America)  \nand fifth in population (after Asia, Africa, Europe, and  \nNorth America). The word America was coined in 1507 by  \ncartographers Martin Waldseemüller and Matthias Ringmann,  \nafter Amerigo Vespucci, who was the first European to  \nsuggest that the lands newly discovered by Europeans were  \nnot India, but a New World unknown to Europeans.",
  "surface": 17840000,
  "timezone": [
    -4,
    -2
  ],
  "demographics": {
    "population": 385742554,
    "largestCities": [
      "São Paulo",
      "Buenos Aires",
      "Rio de Janeiro",
      "Lima",
      "Bogotá"
    ]
  },
  "languages": [
    "spanish",
    "portuguese",
    "english",
    "dutch",
    "french",
    "quechua",
    "guaraní",
    "aimara",
    "mapudungun"
  ],
  "countries": [
    {
      "name": "Argentina",
      "capital": "Buenos Aires",
      "independence": "1816-07-08T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Bolivia",
      "capital": "La Paz",
      "independence": "1825-08-05T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Brazil",
      "capital": "Brasilia",
      "independence": "1822-09-06T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Chile",
      "capital": "Santiago",
      "independence": "1818-02-11T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Colombia",
      "capital": "Bogotá",
      "independence": "1810-07-19T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Ecuador",
      "capital": "Quito",
      "independence": "1809-08-09T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Guyana",
      "capital": "Georgetown",
      "independence": "1966-05-25T23:00:00.000Z",
      "unasur": true
    },
    {
      "name": "Paraguay",
      "capital": "Asunción",
      "independence": "1811-05-13T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Peru",
      "capital": "Lima",
      "independence": "1821-07-27T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Suriname",
      "capital": "Paramaribo",
      "independence": "1975-11-24T23:00:00.000Z",
      "unasur": true
    },
    {
      "name": "Uruguay",
      "capital": "Montevideo",
      "independence": "1825-08-24T23:06:32.000Z",
      "unasur": true
    },
    {
      "name": "Venezuela",
      "capital": "Caracas",
      "independence": "1811-07-04T23:06:32.000Z",
      "unasur": true
    }
  ]
}');

SELECT * FROM public.tjsonb;
```

Then make a double-click on the `payload` cell, it won't react.

Expected behavior: the inline editor is activated and user is able to select, copy/paste and edit the value (it used to work in 6.2.0 version, therefore I consider it as a regression bug).

**Comment by serge-rider** [comment]
It is not a bug. Inline editor makes very little sense for JSON(B).
Why don't you use value edit panel? Double-clicking on json cell shoudl activate it. Or you can atrivate/deactivate it manually.

![image](https://user-images.githubusercontent.com/6398845/71173810-b9624f80-2274-11ea-8154-6bbc56cff8f9.png)

**Comment by flashtheman** [comment]
I confirm, it does not work in version 6.3.0.201912141927.

**Comment by serge-rider** [comment]
Value view panel for jsonb and struct types are broken in 6.3.0.
But we already released hotfix version (you can download latest build or Early Access version from dbeaver.io). And the fix will be  in 6.3.1 of course.
